src/index.html

```diff
@@ -1,13 +1,266 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BIKE</title>
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.png"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
```

Comment:
600 font-weight should also be imported - it used on the page:

![image](https://github.com/mate-academy/layout_landing-page/assets/86923089/9fbf396e-22ca-4452-9221-0ced914218fe)